src/hyperlight_host/src/hypervisor/gdb/mod.rs

```diff
@@ -26,6 +27,7 @@ use event_loop::event_loop_thread;
 use gdbstub::conn::ConnectionExt;
 use gdbstub::stub::GdbStub;
 use gdbstub::target::TargetError;
+pub use hyp_debug::{kvm, mshv, GuestMemoryDebug, GuestVcpuDebug};
```

Comment:
Does this need to be pub?

Reply:
Yes, it is used in each hypervisor driver.

Reply:
Could you use `pub(crate) use hyp_debug::{kvm, mshv, GuestMemoryDebug, GuestVcpuDebug};` instead?

Reply:
+1 on pub(crate)

Reply:
Oh, I see what you meant now.
Good catch!